src/senaite/sync/browser/views.py

```diff
@@ -277,7 +315,7 @@ def update_object_with_data(self, obj, data, domain):
                 logger.error("Could not set field '{}' with value '{}'".format(fieldname, value))
 
         # finally reindex the object
-        obj.reindexObject()
+        self.uids_to_reindex.append([api.get_uid(obj), repr(obj)])
```

Comment:
Why `repr(obj)`?

Reply:
If objects disappear, I thought it would be nice to see what they were. `UID` doesn't help since they are not in DB any more. ID was not enough, since there can be Analyses with the same ID under different AR folders... Could be `str(obj)`  too though, but I trust `repr` more.